### PR TITLE
more texture maps cbt

### PR DESCRIPTION
**Pull request by DniweTamp**

### CODE REVIEW

File: templates/p5_gfd.bt [hunk 1]
```diff
@@ -259,6 +259,20 @@ typedef struct
  TBool32 TMaterialFlags_Flag1            : 1;
 } TMaterialFlags;
 
+typedef struct
+{ 
+    EMaterialMapsTexcoord TMaterialMaps_Filler           : 5<hidden=true>;
```
Comment (TGEnigma) [inline templates/p5_gfd.bt:264]: not an enum, so the member names dont have to prefixed with TMaterialMaps

File: templates/p5_gfd.bt [hunk 2]
```diff
@@ -318,8 +332,8 @@ typedef struct(u32 version)
     }
 
     s16 Field5C;
-    u32 Field6C<format=hex>;
-    u32 Field70<format=hex>;
+    TMaterialMapsTexcoord Field6C;
```
Comment (TGEnigma) [inline templates/p5_gfd.bt:335]: rename the field too

File: templates/p5_gfd.bt [hunk 3]
```diff
@@ -318,8 +332,8 @@ typedef struct(u32 version)
...
+    TMaterialMapsTexcoord Field70;
```
Comment (TGEnigma) [inline templates/p5_gfd.bt:336]: idem

File: templates/p5_gfd.bt [hunk 4]
```diff
@@ -896,11 +910,7 @@ typedef struct
     f32 ClipPlaneFar;
     f32 FieldOfView;
     f32 AspectRatio;
-
-    if ( Version > 0x1104060 )
-    {
-        f32 Field190;
-    }
+    f32 Field190;
```
Comment (TGEnigma) [inline templates/p5_gfd.bt:913]: why did you remove the version check

**Review by TGEnigma — CHANGES_REQUESTED** [review]

File: templates/p5_gfd.bt (hunk 1 above)
Comment (DniweTamp) [reply templates/p5_gfd.bt:264]: so should it just be Filler

File: templates/p5_gfd.bt (hunk 2 above)
Comment (DniweTamp) [reply templates/p5_gfd.bt:335]: to wot

File: templates/p5_gfd.bt (hunk 4 above)
Comment (DniweTamp) [reply templates/p5_gfd.bt:913]: answered on discord lol
i'll bring it back in next commit i guess

File: templates/p5_gfd.bt (hunk 1 above)
Comment (TGEnigma) [reply templates/p5_gfd.bt:264]: yes, same goes for the rest

File: templates/p5_gfd.bt (hunk 2 above)
Comment (TGEnigma) [reply templates/p5_gfd.bt:335]: TexcoordFlags
